mail_activity_done/models/res_users.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018-22 ForgeFlow <http://www.forgeflow.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models, modules
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    @api.model
+    def systray_get_activities(self):
```

Comment:
I get traceback:

`UncaughtPromiseError
Uncaught Promise > Identifying field "id" is lacking a value on ir.model with 'and' identifying mode
Error: Identifying field "id" is lacking a value on ir.model with 'and' identifying mode
    at ModelIndexAnd.findRecord (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10340:159)
    at ModelManager.findFromIdentifyingData (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10377:88)
    at ModelManager._insert (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10471:62)
    at ModelManager._preInsertIdentifyingFieldsFromData (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10490:20)
    at ModelManager.findFromIdentifyingData (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10377:6)
    at ModelManager._insert (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10471:62)
    at ModelField._insertOtherRecord (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10235:35)
    at ModelField._setRelationInsertAndReplace (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10239:71)
    at ModelField.parseAndExecuteCommands (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10219:40)
    at ModelManager._update (https:/customer/web/assets/778166-4f926fb/web.assets_backend.min.js:10512:40)`